host/enclavemanager.c

```diff
@@ -47,6 +48,7 @@ uint32_t oe_push_enclave_instance(oe_enclave_t* enclave)
         {
             if (tmp->enclave == enclave)
             {
+                OE_TRACE_ERROR("the enclave is already in global list\n");
```

Comment:
Start the error message with a capital letter i.e. "The enclave..."

Reply:
addressed